WordPress/Classes/ViewRelated/Post/PostListFilter.m

```diff
@@ -0,0 +1,66 @@
+#import "PostListFilter.h"
+#import "BasePost.h"
+@implementation PostListFilter
+
++ (NSArray *)newPostListFilters
+{
+    return @[
+             [self newPublishedFilter],
+             [self newDraftFilter],
+             [self newScheduledFilter],
+             [self newTrashedFilter]
+             ];
+}
+
++ (instancetype)newPublishedFilter
+{
+    PostListFilter *filter = [PostListFilter new];
+    filter.title = NSLocalizedString(@"Published", @"Title of the published filter. This filter shows a list of posts that the user has published.");
+    filter.statuses = @[PostStatusPublish, PostStatusPrivate];
+    filter.filterType = PostListStatusFilterPublished;
+    filter.predicateForFetchRequest = [NSPredicate predicateWithFormat:@"status IN %@", filter.statuses];
+    return filter;
+}
+
++ (instancetype)newDraftFilter
+{
+    PostListFilter *filter = [PostListFilter new];
+    filter.title = NSLocalizedString(@"Draft", @"Title of the draft filter.  This filter shows a list of draft posts.");
+    filter.statuses = @[PostStatusDraft, PostStatusPending];
```

Comment:
Short answer, we're matching the web version.  I'm guessing the rationale is that pending is a workflow step for a draft on its way to being published. A pending post is not visible, and may require additional edits before its publishable.